frontend/src/documents/DocumentPreview.svelte

```diff
@@ -1,4 +1,5 @@
 <script lang="ts" context="module">
+  const plainTextExtensions = ["csv", "txt", "qfx"];
```

Comment:
Note that `prettier` prefers this all on one line, instead of broken into multiple lines like `imageExtensions`.